### ISSUE TITLE
Add install package here to balance.do since make crashes without that

### ISSUE THREAD

**Issue by zaveczgergo** [korenmiklos/expat-analysis#85]

**Comment by korenmiklos** [comment]
Can you cherry pick the commit?

zaveczgergo <notifications@github.com> ezt írta (időpont: 2021. jan. 28.,
Cs, 17:19):

**Comment by zaveczgergo** [comment]
Do you mean that once I added the code and commited it, can one cherry pick it to other branches?

**Comment by zaveczgergo** [comment]
Added problem: add install reghdfe to balance. do

**Comment by korenmiklos** [comment]
No, I mean we have an install.do somewhere (we should) and you could cherry
pick that. Maybe in code/util?

zaveczgergo <notifications@github.com> ezt írta (időpont: 2021. jan. 28.,
Cs, 17:46):

**Comment by zaveczgergo** [comment]
Oh, ok, I check that.
